apps/landingPage/next.config.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+module.exports = {
+  exportPathMap: function() {
+    return {
+      '/': { page: '/' },
+    };
+  },
+};
```

Comment:
Is this necessary? I don't think it does anything else than what is default?

Reply:
Probably not if its default, going to try it out without

Reply:
Its not working without it